### ISSUE TITLE
Index count on reindexing of canvas/page not updating

### ISSUE THREAD

**Issue by mejackreed** [sul-dlss/exhibits#808]
Split off of #806

**Comment by blalbrit** [comment]
@mejackreed - is this still a needed ticket?

**Comment by mejackreed** [comment]
It would need to be investigated.

**Comment by blalbrit** [comment]
Leaving open then, removing Parker label.

**Comment by mejackreed** [comment]
Should be worth noting that Parker is the only exhibit at this point that uses this type of indexing strategy.

**Comment by blalbrit** [comment]
Yep - good to note. However, it is likely that we will never have another Parker sprint, so this is now backlogged for "exhibits" as a project in general.
